tinker/templates/faculty-bios/home.html

```diff
@@ -109,57 +109,61 @@
                 {% if show_create %}
                     <a href="{{ url_for('FacultyBiosView:new') }}" class="btn btn-primary">Create New Bio</a>
                     {% if show_special_admin_view %}
-                     <a href="{{ url_for('FacultyBiosView:get_faculty_bio_csv') }}" class="btn btn-primary">Export csv file</a>
+                        <a href="{{ url_for('FacultyBiosView:get_faculty_bio_csv') }}" class="btn btn-primary">Export csv file</a>
                     {% endif %}
                     <hr/>
                 {% endif %}
-                {% if show_special_admin_view %}
-                    <div>
-                        <label><h3 class="first-subtitle">View:</h3>
-                            <select class="horizontal-scroll" size="{{ all_schools|length }}" id="school-selector"
-                                    multiple>
-                                {% for schools in all_schools %}
-                                    {% for class_name, school_name in schools.items() %}
-                                        <option value="{{ class_name }}"
-                                                selected="selected">{{ school_name }}</option>
-                                    {% endfor %}
+                <div>
+                    <label><h3 class="first-subtitle">View:</h3>
+                        <select class="horizontal-scroll" size="{{ all_schools|length }}" id="school-selector"
+                                multiple>
+                            {% for schools in all_schools %}
+                                {% for class_name, school_name in schools.items() %}
+                                    <option value="{{ class_name }}" {{ 'selected' if class_name in class_list or show_special_admin_view }}>{{ school_name }}</option>
```

Comment:
How does the selected work if you have multiple?

Reply:
It's a multiple select so if you have multiple, or you are an admin and can see all schools, it can load all of them as necessary.